lxl-web/src/colors.css

```diff
@@ -10,17 +10,33 @@ see https://tailwindcss.com/docs/customizing-colors#using-css-variables */
 	--color-beige: 248 246 241;
 	--color-pink: 236 214 192;
 
+	/* Fundament */
 	--color-primary: var(--color-brown);
 	--color-accent-dark: var(--color-dark-green);
 	--color-accent-light: var(--color-light-green);
-	--color-background: var(--color-beige);
+	--color-subtle: var(--color-beige);
 	--color-highlight: var(--color-pink);
+
+	/* Text colors */
+	--text-primary: var(--color-primary);
+	--text-primary-inv: var(--color-white);
+	--text-link: var(--color-accent-dark);
+
+	/* Bg colors */
+	--bg-main: var(--color-subtle);
+	--bg-head: var(--color-accent-light);
+	--bg-positive: var(--color-accent-light);
```

Comment:
When is `--bg-positive` intended to be used? For active states on elements?

Reply:
I interpret it as the bg of an 'active' element such as (in this case) a filter pill.
It's a figma variable, there are a bunch:
<img width="446" alt="Skärmavbild 2024-02-14 kl  09 07 22" src="https://github.com/libris/lxlviewer/assets/22232928/ab24f4fa-f5da-454a-b5a6-2df221681671">

I'm not sure if we should add all these to the project. The upside is it adds flexibility (for whitelabelling etc) and easy styling if they are used often. The downside is we'll eventually get a lot of variables, harder to maintain. I've just added those i've encoutered so far. But i'm up for discussion :)